popup.js

```diff
@@ -201,16 +201,18 @@ function runThePopup() {
     }
   });
   
-  
   chrome.tabs.query({
       active: true,
       currentWindow: true
-  }, loadTabData);
+  }, onTabs);
   
+  function onTabs(tabs) {
+	 chrome.storage.sync.get(['videoID'], function(result) {
+         	loadTabData(tabs, result.videoid);
+	 }); 
+  }
   
-  function loadTabData(tabs) {
-    //set current videoID
-    currentVideoID = getYouTubeVideoID(tabs[0].url);
+  function loadTabData(tabs, currentVideoID) {
```

Comment:
Is `tabs` still needed here?

Reply:
Currently Yes as tabs[0].id is used for isInfoFound